### ISSUE TITLE
Scroll not working with automated test using selenium and wdio

### ISSUE THREAD

**Issue by nuria-gs** [KingSora/OverlayScrollbars#227]
**Describe the bug**
Hi, 
First of all congratulations for the package, it's amazing!!!

I have used it on a project where we have automated tests using selenium and webdriverio (V.4). The issue is that the application is not scrolling. It works fine when doing it manually but not running the automated tests. Webdriver use its own commands for scrolling controlling the browser scroll and window, and it seems usual scroll commands do not work with the OverlayScrollbars. 
Not sure if this is supported by the package or if someone has used it with automated tests, as I'm having troubles to make it work.

Thanks

**Comment by nuria-gs** [comment]
Yes, the have it but it's not working with the overlayScrollbars, not sure if it's because it's replacing the native scrollbars and it the scroll needs to be handled from overlayScrollbars.

http://v4.webdriver.io/v3.4/api/utility/scroll.html

**Comment by KingSora** [comment]
This is happening because `OverlayScrollbars` changes the element which is responsible for scrolling.
Lets say you apply the plugin to the body element. 

Previously you could do something like `window.scrollTo(x, y)` but thats not possbile anymore because the plugin took the scrolling "power" from the window / html / body element and shifted it to a different element. I assume that the framework you're using is just sending commands to the window element in order to scroll, which won't work once `OverlayScrollbars` is applied.

To fix it, you have to tell your framework to send the coordinates either to a different element, or you can execute your own javascript (on the webpage), to take care of that by yourself.

**Comment by KingSora** [comment]
@nuria-gs any news here? Did you manage to implement it based on my comment?

**Comment by nuria-gs** [comment]
Hi KingSora, 

Sorry for the delay I thought I had answered. 

I made a workaround avoiding having to scroll on the test as scrolling wasn't the aim of the feature. I didn't implement the scrolling action, as we use webdriverio which it's a third testing framework to ouse selenium. Also we are using an old version of webdriverio, so I'll wait until to upgrade the version to see if it works with the new one, and in the meanwhile we are preventing to have to scroll on the tests.
